### PR TITLE
logging: adds Heroku logging endpoint support

### PR DESCRIPTION
**Pull request by mccurdyc**
## Relevant (Referenced) Link(s)

* [Fastly API Docs](https://docs.fastly.com/api/logging#api-section-logging_heroku)
* [fastly/go-fastly](https://github.com/fastly/go-fastly/blob/c2c2c62210800daab641161412fbe210004aac0a/fastly/heroku.go)

## Validation

```
$ make test
```

```
git clone git@github.com:mccurdyc/cli.git /tmp/cli && \
(
    cd /tmp/cli && \
    git checkout mccurdyc/logging-heroku && \
    make test && \
    rm -rf /tmp/cli \
)
```

### CODE REVIEW

File: pkg/logging/heroku/create.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+// NewCreateCommand returns a usable command registered under the parent.
+func NewCreateCommand(parent common.Registerer, globals *config.Data) *CreateCommand {
+	var c CreateCommand
+
+	// Without this, parsing flags leads to panics.
+	// This is because if we try to scan directly into c.Input.<Field> without first
+	// creating a valid reference, flags will attempt to be scanned into null pointers.
+	// This will result in sending the zero value values for unset flags which is acceptable
+	// for Creates, but not Updates.
```
Comment (mccurdyc) [inline pkg/logging/heroku/create.go:29]: Good? Necessary?

**Comment by pteichman** [comment]
Conflict fix is https://github.com/fastly/cli/commit/28379725d7d2f4032a01041894f569b4108293a8

I squashed the first two commits together since they were both necessary to get a clean build.

**Comment by mccurdyc** [comment]
Force-pushed Peter's commit.

**Comment by mccurdyc** [comment]
Force-pushed rebasing master.

**Comment by mccurdyc** [comment]
Force-pushed squashing into single commit.

**Comment by mccurdyc** [comment]
Force-pushed squashed commit including stronger tests.

**Comment by mccurdyc** [comment]
Force-pushed rebasing master.

**Pull request merged**